### ISSUE TITLE
Link Dialog Modal Close Button is not using the specified icon library 

### ISSUE THREAD

**Issue by muellercornelius** [iliyaZelenko/tiptap-vuetify#130]
When using the mdi Icon Package the Close Icon is not using the specified icon library. 

![image](https://user-images.githubusercontent.com/26606825/72682889-6f1df980-3ad2-11ea-91b7-85682a8c4ce9.png)

The other Icons are set and displayed correctly.

```
Vue.use(TiptapVuetifyPlugin, {
  vuetify,
  iconsGroup: 'mdi'
})
```

**Comment by constructivetim** [comment]
I am also experiencing this.  did you find a work around?

**Comment by iliyaZelenko** [comment]
Fixed in v2.12.2.
